x-pack/plugins/cases/server/saved_object_types/migrations/cases.ts

```diff
@@ -131,6 +134,17 @@ export const addAssignees = (
   return { ...doc, attributes: { ...doc.attributes, assignees }, references: doc.references ?? [] };
 };
 
+export const convertSeverity = (
+  doc: SavedObjectUnsanitizedDoc<CaseAttributes>
+): SavedObjectSanitizedDoc<Omit<CaseAttributes, 'severity'> & { severity: ESCaseSeverity }> => {
+  const severity = SEVERITY_EXTERNAL_TO_ESMODEL[doc.attributes.severity] ?? ESCaseSeverity.LOW;
```

Comment:
Being paranoid here but to avoid surprises do you mind if we do `doc.attributes.severity ?? CaseSeverity.LOW`?

Reply:
Nevermind `SEVERITY_EXTERNAL_TO_ESMODEL[undefined] === undefined` so it will fallback to `ESCaseSeverity.LOW`